lib/tasks/lint.rake

```diff
@@ -1,5 +0,0 @@
-desc "Run govuk-lint on all files"
-task "lint" do
-  sh "govuk-lint-ruby --format clang"
-  sh "govuk-lint-sass app/assets/stylesheets"
-end
```

Comment:
What was the reason for deleting this?

Reply:
It was causing all rake tasks to break on the PaaS, I have a vague intention of possibly fixing it but I'm also aware that I'm the only person who has ever used this repo so don't want to spend too much time on such things

Reply:
`govuk-lint` is also deprecated and we should be using https://github.com/alphagov/rubocop-govuk instead.